File: admin/src/app/identity-management/models/users-dto.ts
export class UsersDto {
    fullName: string;
    email: string;
    password: string;
    profilePhoto: string;
    authProvider: string;
    roleCode: string;
    roleName: string;
    volunteer: boolean;
    isActive: boolean;
    deletedBy: string;
    deletedDate: string;
    isDeleted: boolean;
    updatedDate?: string;
    updatedBy?: string;
    createdDate: string;
    createdBy?: string;
    id: string;
  }

File: aspnet-core/VibeQuest.Dto/UsersDto.cs
using Microsoft.AspNetCore.Http;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace VibeQuest.Dto
{
    public class UsersDto : EntityDto<Guid>
    {
        public string FullName { get; set; }

        public string Email { get; set; }

        public string Password { get; set; }

        public string ProfilePhoto { get; set; }

        public string AuthProvider { get; set; }

        public bool IsActive { get; set; }
        public string RoleCode { get; set; }
        public string RoleName { get; set; }
        public bool Volunteer { get; set; }
        public string ProfilePhotoFullPath { get; set; }
        public string AppleUserId { get; set; }
        public bool? EmailSubscribed { get; set; }
        public bool? OptOutPassword { get; set; }
        public DateTime? PasswordResetDate { get; set; }
    }

    public class ResetPasswordWebDto
    {
        public int Id { get; set; }
        public string Code { get; set; }
        public string NewPassword { get; set; }
    }
}
